public/controllers/management/components/management/configuration/edit-configuration/edit-configuration.js

```diff
@@ -109,8 +105,8 @@ class WzEditConfiguration extends Component {
         severity: UI_ERROR_SEVERITIES.BUSINESS,
         error: {
           error: error,
-          message: errorMessage || error,
-          title: `${error.name}: Mitre alerts could not be fetched`,
+          message: errorMessage || error,
+          title: "The configuration couldn't be saved.",
```

Comment:
**nitpick**: The file is saved, but it could be some validation errors. The message could be confusing.